usb/backend/libusb0.py

```diff
@@ -619,27 +619,47 @@ def reset_device(self, dev_handle):
 
     @methodtrace(_logger)
     def is_kernel_driver_active(self, dev_handle, intf):
+        # libusb0's usb_get_driver_np() is platform specific / not portable.
+        # This function checks what kernel module handles dev_handle/intf.
+        # Various implementations may return system specific values - for
+        # instance Linux may return errno.ENODATA not present on FreeBSD.
+        # On FreeBSD successul call returns 0 otherwise -1.
+
```

Comment:
This goes in a bit too much detail, but also isn't clear enough.  Remove all but the first lines of this comment...

Reply:
```
622         # libusb0's usb_get_driver_np() is platform specific / not portable.
623         # This function checks what kernel module handles dev_handle/intf.
624         # Various implementations may return system specific values.
```
?